commons/src/main/java/fr/free/nrw/commons/upload/MwVolleyApi.java

```diff
@@ -18,23 +19,75 @@
 
 
 import java.io.UnsupportedEncodingException;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
 
 public class MwVolleyApi {
 
     private static RequestQueue REQUEST_QUEUE;
     private static final Gson GSON = new GsonBuilder().create();
     private Context context;
+    private static String coordsLog;
```

Comment:
I can make this an instance variable if I remove its usage in the static method, which doesn't really have any adverse consequences since it's only used in a debugging log that isn't really needed at this stage (so I did that).